inc/abstracttarget.class.php

```diff
@@ -195,6 +195,10 @@ abstract protected function getTargetTemplate(array $data): int;
    const LOCATION_RULE_SPECIFIC = 2;
    const LOCATION_RULE_ANSWER = 3;
 
+   const VALIDATION_RULE_NONE = 1;
+   const VALIDATION_RULE_SPECIFIC_USER_OR_GROUP = 2;
+   const VALIDATION_RULE_ANSWER_USER = 3;
```

Comment:
I think the columns should be renamed : we don't know if the validation is related to form answer or ticket. In the future, we may have headache to maintain and debug.

- TICKET_VALIDATION_RULE_NONE
- TICKET_VALIDATION_RULE_SPECIFIC_USER_OR_GROUP
- TICKET_VALIDATION_RULE_ANSWER_USER